web/cypress/e2e/onboarding-as-foreign-business.spec.ts

```diff
@@ -0,0 +1,164 @@
+import {
+  randomElementFromArray,
+  setMobileViewport,
+} from "@businessnjgovnavigator/cypress/support/helpers/helpers";
+import { onOnboardingPageNexusBusiness } from "@businessnjgovnavigator/cypress/support/page_objects/onboardingPageNew";
+import {
+  Industries,
+  Industry,
+  ResidentialConstructionType,
+  carServiceOptions,
+  randomInt,
+} from "@businessnjgovnavigator/shared/";
+
+const enabledIndustries = Industries.filter((element: Industry) => {
+  return element.isEnabled;
+});
+
+describe("Onboarding for all industries when starting a business [feature] [all] [group4]", () => {
+  describe("Desktop", () => {
+    beforeEach(() => {
+      cy.loginByCognitoApi();
+    });
+
+    for (const industry of enabledIndustries) {
+      it(`Onboarding for ${industry.name}`, () => {
+        cy.url().should("include", "onboarding?page=1");
+        onOnboardingPageNexusBusiness.selectBusinessPersonaRadio("FOREIGN");
+        onOnboardingPageNexusBusiness.getBusinessPersonaRadio("FOREIGN").should("be.checked");
+        onOnboardingPageNexusBusiness.clickNext();
+
+        cy.url().should("include", "onboarding?page=2");
+        onOnboardingPageNexusBusiness.selectEmployeeOrContractorInNJCheckBox("employeeOrContractorInNJ");
```

Comment:
pebble - since the name of this function is specific, I don't think it's necessary to pass in the argument. `onOnboardingPageNexusBusiness.selectEmployeeOrContractorInNJCheckBox( )` should suffice.